python_dependency_verifier/tests/unittests.py

```diff
@@ -0,0 +1,44 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+############
+
+import unittest
+from python_dependency_verifier import python_dependency_verifier
+
+
+class TestDependencies(unittest.TestCase):
+
+    def runTest(self):
+        pass
+
+    def check_file(self):
+        return self.assertEqual(
+            python_dependency_verifier.check_dependencies_file(
+                "../../setup.py", "cloudify.*"),
```

Comment:
You're using the main setup.py file. What if it changed and you add another dependency?
You should probably create a resources directory and put a mock setup.py file there.